specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTranscriptJobsTaskSubmitRequest.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
The names of these files are just too long. Too many things: ConversationTranscription, JobsTask
We can shorten this by a lot. e.g. `SuccessfulConversationSummarySubmit.json` is enough.

Reply:
Agreed.